src/js/ripple/remote.js

```diff
@@ -1615,10 +1542,17 @@ Remote.parseBinaryLedgerData = function(ledgerData) {
 Remote.prototype.requestTransactionHistory =
 Remote.prototype.requestTxHistory = function(start, callback) {
   // XXX Does this require the server to be trusted?
-  //utils.assert(this.trusted);
+  // utils.assert(this.trusted);
 
   var request = new Request(this, 'tx_history');
 
+  if (typeof start === 'object') {
+    start = start.start;
```

Comment:
I see what's going on now. I think the code should be oriented toward the non-deprecated usage i.e. 

    function(opts, callback)

In the else block

    opts = {start: opts}

and

    request.message.start = opts.start;